src/HttpHdrSc.cc

```diff
@@ -38,7 +38,8 @@ LookupTable<http_hdr_sc_type> scLookupTable(SC_ENUM_END, ScAttrs);
 std::vector<HttpHeaderFieldStat> scHeaderStats(SC_ENUM_END);
 
 // used when iterating over flags
-http_hdr_sc_type &operator++ (http_hdr_sc_type &aHeader)
+http_hdr_sc_type &
+operator++ (http_hdr_sc_type &aHeader)
```

Comment:
Avoid formatting-only change?

Reply:
Good catch. It had crept in when I tried to make it static. Reverting